src/push-state-tree.js

```diff
@@ -751,18 +861,16 @@ if (typeof PST_NO_OLD_IE == 'undefined'
 
     location.hash = uri;
 
-    document.title = title;
-    lastTitle = title;
-
     return this;
   };
 
-  PushStateTree.prototype.replaceState = function (state, title, uri) {
-    title = title || document.title;
-    uri = uri || '';
-    if (lastTitle !== null) {
-      document.title = lastTitle;
+  PushStateTree.prototype.replaceState = function (uri) {
+    if (typeof arguments[2] === 'string') {
```

Comment:
why not just add the parameter to the arglist? seems unnecessary to use `arguments[2]`